infrastructure/kzg/src/main/java/tech/pegasys/teku/kzg/KZG.java

```diff
@@ -65,6 +70,24 @@ public KZGProof computeBlobKzgProof(final Bytes blob, final KZGCommitment kzgCom
             throws KZGException {
           return KZGProof.fromBytesCompressed(Bytes48.ZERO);
         }
+
+        @Override
+        public List<KZGCellAndProof> computeCellsAndProofs(Bytes blob) {
+          throw new RuntimeException("Not implemented");
+        }
+
+        @Override
+        public boolean verifyCellProofBatch(
+            List<KZGCommitment> commitments,
+            List<KZGCellWithColumnId> cellWithIDs,
+            List<KZGProof> proofs) {
+          return false;
```

Comment:
I'd make old (deneb) boolean methods return false in default too. If accidentally NOOP leaks in production we'd better fail all checks

Reply:
That makes it harder for unit tests. Most of the time we want NOOP to do nothing and stick to the "standard" flow.